ebuild-writing/using-eclasses/text.xml

```diff
@@ -25,6 +25,14 @@ illegal (except where the inheritance criteria are cache-constant <d/> see <uri
 link="::general-concepts/portage-cache"/>).
 </p>
 
+<p>
+When using <c>inherit</c>, sort the arguments (eclasses) alphabetically, except
+for where the phases exported by an eclass are affected by subsequent arguments,
+e.g. <c>multilib-minimal.eclass</c> mentions in its
+<uri link="::eclass-reference/multilib-minimal.eclass/">documentation</uri>
+that it should be inherited last because it overrides most phases.
```

Comment:
This sentence is too long. Full stop after "arguments", then start a new sentence "For example, ...".

Reply:
Done.